packages/ui-tooltip/src/Tooltip/README.md

```diff
@@ -59,74 +59,121 @@ type: example
+            </Tooltip>
+          </p>
+          <Checkbox
+            label="show tooltip?"
+            variant="toggle"
+            value="toggled"
+            onChange={(event) => {
+              this.setState({ isShowingContent: event.target.checked })
+            }}
+          />
+        </>
+      )
     }
   }
 
-  render () {
+  render(<Example />)
+  ```
+
+- ```js
+  const Example = () => {
+    const [isShowingContent, setIsShowingContent] = useState(false)
+
     return (
       <>
         <p>
-        <Tooltip
-          renderTip="Hello. I'm a tool tip"
-          isShowingContent={this.state.isShowingContent}
-          onShowContent={(e) => {
-            console.log("expecting to show tooltip")
-          }}
-          onHideContent={(e) => {
-            console.log("expecting to hide tooltip")
-          }}
-        >
-          <Link href="#">This link has a tooltip</Link>
-        </Tooltip>
+          <Tooltip
+            renderTip="Hello. I'm a tool tip"
+            isShowingContent={isShowingContent}
+            onShowContent={(e) => {
+              console.log('expecting to show tooltip')
+            }}
+            onHideContent={(e) => {
+              console.log('expecting to hide tooltip')
+            }}
+          >
+            <Link href="#">This link has a tooltip</Link>
+          </Tooltip>
         </p>
         <Checkbox
           label="show tooltip?"
           variant="toggle"
           value="toggled"
           onChange={(event) => {
-            this.setState({isShowingContent: event.target.checked})
+            setIsShowingContent(event.target.checked)
           }}
         />
-    </>
-  )
+      </>
+    )
   }
-}
 
-render(<Example />)
-```
+  render(<Example />)
+  ```
 
 ### Custom elements as renderTrigger
 
 Popover and Tooltip attach mouse and focus event listeners to their render trigger components via props. These need to be propagated to the component for the listeners to work:
 
-```js
----
-  type: example
----
-class MyComponent extends React.Component {
-  constructor(props) {
-    super(props)
-    this.ref = React.createRef()
-  }
-  render() {
-    //  Spread the props to the underlying DOM element
-    return <div {...this.props} ref={this.ref} style={{width:"10rem"}}>My custom component</div>
+- ```js
+  class MyComponent extends React.Component {
+    constructor(props) {
+      super(props)
+      this.ref = React.createRef()
+    }
+    render() {
+      //  Spread the props to the underlying DOM element
+      return (
+        <div {...this.props} ref={this.ref} style={{ width: '10rem' }}>
+          My custom component
+        </div>
+      )
+    }
   }
-}
-<Tooltip renderTip="Tooltip text to display" >
-  <MyComponent />
-</Tooltip>
-```
+  ;<Tooltip renderTip="Tooltip text to display">
+    <MyComponent />
+  </Tooltip>
+  ```
+
+- ```js
+  const MyComponent = React.forwardRef((props, ref) => {
```

Comment:
I'm not sure where the imports for the docs examples are defined, but if you could point out where they are defined, I can replace this with just `forwardRef` instead.

Reply:
They are defined in https://github.com/instructure/instructure-ui/blob/master/packages/__docs__/globals.js , feel free to add new imports from React here

Reply:
Done, thanks!